README.md

```diff
@@ -78,6 +78,10 @@ GINIVision.setConfiguration(giniConfiguration)
 
 We are providing example apps for Swift and Objective-C. These apps demonstrate how to integrate the Gini Vision Library with the Screen API and Component API. The Gini API SDK is used to analyze the photos of documents. To run the example projects, clone the repo and run `pod install` from the Example or ExampleObjC directory first.
 
+## Swift versions
+
+The Gini Vision Library is entirely written in Swift 2 and needs to be compiled with Xcode versions 7.X. However, you can find both Swift 2.3 and Swift 3 support in different branches within the repository. Please refer to the [Branches](#Branches) section later in this document. 
```

Comment:
Add a note on how to use a Swift branch as seen in the branch `swift-2.3` already.